### PR TITLE
Update jupyter notebook

### PR DESCRIPTION
**Pull request by AyushShahh**
### Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: <https://github.com/github/explore/blob/main/CONTRIBUTING.md>.
- [x] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [x] Suggesting edits to an existing topic or collection
  - [ ] Curating a new topic or collection
  - [ ] Something that does not neatly fit into the binary options above

---

### Editing an existing topic or collection

I'm suggesting these edits to an existing topic or collection:
- [x] Image (and my file is `*.png`, square, dimensions 288x288, size <= 75 kB)
- [x] Content (and my changes are in `index.md`)

> Jupyter notebook was previously IPython notebook. Added aliases, related topics and Wikipedia link. Also updated logo

### Curating a new topic or collection

- [ ] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [ ] My folder contains a `*.png` image (if applicable) and `index.md`
- [ ] All required fields in my `index.md` conform to the Style Guide and API docs: <https://github.com/github/explore/tree/main/docs>

> Please replace this line with an explanation of why you think this topic or collection should be curated.

### Something that does not neatly fit into the binary options above

- [ ] My suggested edits are not about an existing topic or collection, or at least not a single one
- [ ] My suggested edits are not about curating a new topic or collection, or at least not a single one
- [ ] My suggested edits conform to the Style Guide and API docs: https://github.com/github/explore/tree/main/docs

> Please replace this line with an explanation of your proposed changes.

---

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

File: topics/jupyter-notebook/index.md [hunk 1]
```diff
@@ -1,13 +1,15 @@
 ---
-created_by: Jupyter Developers
+aliases: notebook, jupyter, ipython-notebook, ipynb, ipynb-jupyter-notebook, ipynb-notebook
```
Comment (smashwilson) [inline topics/jupyter-notebook/index.md:2]: I think `notebook` is too broad to include as an alias here. Looking through [the topic](https://github.com/topics/notebook), I see repositories for plenty of non-Jupyter projects, like [zadam/trilium](https://github.com/zadam/trilium).

The rest look good though 👍🏻 

```suggestion
aliases: jupyter, ipython-notebook, ipynb, ipynb-jupyter-notebook, ipynb-notebook
```

File: topics/jupyter-notebook/index.md [hunk 2]
```diff
@@ -1,13 +1,15 @@
...
-related: jupyter
+related: ipython, python, machine-learning, data-visualization, kernel
 short_description: Jupyter Notebooks are documents that contain live code, visualizations, and narrative text.
 topic: jupyter-notebook
 url: https://jupyter.org/
+wikipedia_url: https://en.wikipedia.org/wiki/Project_Jupyter#Jupyter_Notebook
 ---
 
-The Jupyter Notebook is a language-agnostic HTML notebook application for Project Jupyter. Jupyter notebooks are documents that allow for creating and sharing live code, equations, visualizations, and narrative text together. People use them for data cleaning and transformation, numerical simulation, statistical modeling, data visualization, machine learning, and much more. 
+The Jupyter Notebook, previously known as the IPython Notebook is a language-agnostic HTML notebook application for Project Jupyter. Jupyter notebooks are documents that allow for creating and sharing live code, equations, visualizations, and narrative text together. People use them for data cleaning and transformation, numerical simulation, statistical modeling, data visualization, machine learning, and much more.
```
Comment (smashwilson) [inline topics/jupyter-notebook/index.md:15]: Tiny typographic suggestion while I'm in here 😇 

```suggestion
The Jupyter Notebook, previously known as the IPython Notebook, is a language-agnostic HTML notebook application for Project Jupyter. Jupyter notebooks are documents that allow for creating and sharing live code, equations, visualizations, and narrative text together. People use them for data cleaning and transformation, numerical simulation, statistical modeling, data visualization, machine learning, and much more.
```

**Review by smashwilson — CHANGES_REQUESTED** [review]

File: topics/jupyter-notebook/index.md (hunk 1 above)
Comment (AyushShahh) [reply topics/jupyter-notebook/index.md:2]: Thanks, yes I agree

File: topics/jupyter-notebook/index.md (hunk 2 above)
Comment (AyushShahh) [reply topics/jupyter-notebook/index.md:15]: I'll apply the change

**Review by smashwilson — APPROVED** [review]
Perfect. Thanks!

**Pull request merged**